### DISCUSSION TITLE
Latexify with SEI options

### DISCUSSION THREAD

**Discussion by sklein-tu** [pybamm-team/PyBaMM#1873] [Q&A] [answered]
Hello,

I wanted to use the latexify function with the SEI options, but I get a TypeError:

```
options = {"SEI" : "reaction limited"}
model = pybamm.lithium_ion.DFN(options = options)
model.latexify()
```

TypeError: bad operand type for unary -: 'str'

Thank you!

**Comment by brosaplanella** [comment]
The latexify equation is still not designed to work with SEI options directly, so it might need a bit of tweaking (maybe printing the equations one by one). @priyanshuone6, is there any workaround for this?

**Comment by actualpriyanshu** [comment, marked as answer]
This will be fixed in the next release (See #1885).
For now, you can pip install PyBaMM from develop after the pull request is merged (`pip install -q git+https://github.com/pybamm-team/pybamm.git@develop`)

This is what the equations will look like.

![dfn_sei](https://user-images.githubusercontent.com/64051212/148519446-abc8f881-e82f-4e23-939d-9c5ffea8982f.png)
